src/components/PageHeader/PageHeader.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PageHeader from './PageHeader.jsx';
+
+<Meta title="PageHeader" component={PageHeader} />
+
+# PageHeader
+
+Very large text that can be used to present the title and short additional intro text for a non-article page.
```

Comment:
Nitpick but imo 'Very large text' isn't particularly useful info, the 2nd half of this sentence seems more important